modules/global-config/src/main/java/module-info.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+/**
+ * This module handles the global configuration of the Mule Runtime.
+ *
+ * @moduleGraph
+ * @since 4.5
+ */
+module org.mule.runtime.global.config {
+
+  requires org.mule.runtime.api;
+  requires org.mule.runtime.api.annotations;
+  requires org.mule.runtime.container;
+  requires org.mule.runtime.core;
+  requires org.mule.runtime.maven.client.api;
+  requires config;
```

Comment:
what is this?

Reply:
It's from [typesafe](https://github.com/mulesoft/mule/blob/master/modules/global-config/pom.xml#L26)